.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+docs/style-guide.rst @fulldecent
```

Comment:
Github docs say we should be able to see an icon indicating that fulldecent is the code owner. It's supposed to look like this:
![codeowners example](https://docs.github.com/assets/cb-28017/images/help/repository/code-owner-for-a-file.png)

I don't see that when I visit https://github.com/ethereum/solidity/blob/addcodeowners/docs/style-guide.rst. I wonder if it just does not show up for PRs (the docs say that it works per branch though) or if it's because it's a RST file with this weird preview that does not let you view source.

Reply:
I wonder if it might be because of this thing, mentioned in an article I found ([Owning Your CODEOWNERS File](https://medium.com/expedia-group-tech/owning-your-codeowners-file-332e288c1d12)):
> It is not enough to write out a username ( or GitHub Team ) in a `CODEOWNERS` file. The *Collaborators & teams* section of the Repository’s Settings MUST contain everyone mentioned in the `CODEOWNERS `file. GitHub’s docs appear persnickety about this as well,